recipes-sota/aktualizr/aktualizr-auto-prov.bb

```diff
@@ -6,7 +6,8 @@ LICENSE = "MPL-2.0"
 LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MPL-2.0;md5=815ca599c9df247a0c7f619bab123dad"
 
 DEPENDS = "aktualizr-native zip-native"
-RDEPENDS_${PN} = "aktualizr"
+RDEPENDS_${PN}_append = "${@' aktualizr-auto-prov-creds' if d.getVar('SOTA_DEPLOY_CREDENTIALS', True) == '1' else ''}"
+RDEPENDS_${PN} = " aktualizr"
```

Comment:
Do the other prov recipes also need the explicit `RDEPENDS` on `aktualizr`? Might not hurt just to be consistent.

Reply:
Yes, you're right. But I'd rather delete if from here. I believe we've already had a similar discussion, and my point was that custom aktualizr-based projects might use the same provisioning recipes without pulling aktualizr itself. I can't remember if you had objections though.

Reply:
No objection, go ahead and delete it if you'd like.